### PR TITLE
Merge 8.1.x to develop 14/03/23

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Merge 8.1.x to develop 14/03/23
| Type?             | improvement
| Category?         | ME
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI green and campaign green https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/4418129648
| Fixed ticket?     | ~
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]
Approved for UI tests

File: controllers/front/SitemapController.php [hunk 1]
```diff
@@ -91,7 +69,27 @@ public function initContent()
             true
         );
 
-        $this->context->smarty->assign('urls', $urls);
+        /*
+         * Backward compatibility with older themes.
+         * This should be removed as soon as possible, because $pages variable is overwriting
+         * our global template variable assigned in FrontController.
+         */
+        $this->context->smarty->assign(
```
Comment (FabienPapet) [inline controllers/front/SitemapController.php:77]: Can we get rid of this for 9.0, so we don't have to merge it ?

Comment (jolelievre) [reply controllers/front/SitemapController.php:77]: I don't know what this is used for, but anyway it will not be removed in a merge PR

**Review by matthieu-rolland — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Review by 0x346e3730 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]
LGTM for UI tests

**Comment by hibatallahAouadni** [comment]
Hello @jolelievre 

I see all checks are green and UI tests too, is it enough or shall we test it? 

Thanks!

**Pull request merged**